library/test/src/tests.rs

```diff
@@ -63,6 +63,8 @@ fn one_ignored_one_unignored_test() -> Vec<TestDescAndFn> {
                 name: StaticTestName("1"),
                 ignore: true,
                 ignore_message: None,
+                #[cfg(not(bootstrap))]
+                location_info: "src\\lib.rs:10:10: 10:20",
```

Comment:
If these names in tests are never actually used maybe better to set them to empty strings.

Reply:
done.